admin/templates/util/node_preprint_pagination.html

```diff
@@ -0,0 +1,146 @@
+{% load node_extras %}
+{% block content %}
+
+    <div class="pagination pagination-lg">
+        <span>
+            {% if items_paginator.has_previous %}
+                <a href="?{{resource_type}}_page=1{{ current_other_param }}"
```

Comment:
Nice, I like how you're passing in current_other_param to keep track of the other resource's query param